Index: src/use_case/add_food/AddFoodOutputData.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package use_case.add_food;\r\n\r\npublic class AddFoodOutputData {\r\n    private final String output;\r\n\r\n    public AddFoodOutputData(String output){\r\n        this.output = output;\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/use_case/add_food/AddFoodOutputData.java b/src/use_case/add_food/AddFoodOutputData.java
--- a/src/use_case/add_food/AddFoodOutputData.java	
+++ b/src/use_case/add_food/AddFoodOutputData.java	
@@ -1,9 +1,13 @@
 package use_case.add_food;
 
+import java.util.ArrayList;
+
 public class AddFoodOutputData {
     private final String output;
 
     public AddFoodOutputData(String output){
         this.output = output;
     }
+
+
 }
